eclide/RepositoryFilterView.cpp

```diff
@@ -463,6 +463,20 @@ void CRepositoryFilterView::DoRefreshState()
     {
         m_search.EnableWindow(true);
     }
+
+    CString serverIP = GetIConfig(QUERYBUILDER_CFG)->Get(GLOBAL_SERVER_IP);
+    if (!serverIP.GetLength()) {
```

Comment:
I am surprised this works as you will still have a Global Server IP while in local repo mode?
`IsLocalRepositoryEnabled` would be a better test.

Reply:
IsLocalRepositoryEnabled checks if g_EnableCompiler is true. But the grayed out functionalities are for only where there is a server with an IP address.

Reply:
I stand corrected!